src/content/self-hosted/administration/configuration.mdx

```diff
@@ -47,6 +47,26 @@ subdomain: "example.com"
 
 After installation, we recommend that you create a DNS entry for `*.$SUBDOMAIN`, pointing to the public address of your load balancer.
 
+### publicOverride
```

Comment:
Can we have a guide somewhere on how to use this? Changing the publicOverride involves also setting up a new certificate right? Is that only for the main entry point or does it also apply to registry, buildkit, etc?